Cilicon/Provisioner/GitLab Runner/GitLabRunnerProvisioner.swift

```diff
@@ -11,25 +11,47 @@ class GitLabRunnerProvisioner: Provisioner {
     func provision(bundle: VMBundle, sshClient: SSHClient) async throws {
         var downloadCommands: [String] = []
 
+        await SSHLogger.shared.log(string: "Copying config.toml to VM".magentaBold)
```

Comment:
Kind-of. We're creating the `config.toml` file in the correct directory on the VM so that it can be read by the GitLab Runner executable. 

Adjusted the log messages to `Configuring GitLab Runner...` and `Successfully configured GitLab Runner`